api/proto/teleport/legacy/types/types.proto

```diff
@@ -5306,44 +5306,38 @@ message OktaAssignmentSpecV1 {
     (gogoproto.stdtime) = true,
     (gogoproto.nullable) = true
```

Comment:
unrelated: why do we need the distinction between an unset CleanupTime and a zero CleanupTime?

Reply:
I'll adjust this as well. I initially did this because I thought using `nil` was the better way to make this "optional," but as I work with this I realize I don't like it. :-)